app/models/user.rb

```diff
@@ -114,6 +114,11 @@ class User < ActiveRecord::Base
   # will be removed along with /greenfield
   has_many :greenfield_posts, through: :assets
 
+  include Rakismet::Model
+  rakismet_attrs  author: proc { display_name },
+                  author_email: proc { email },
+                  user_ip: proc { current_login_ip }
```

Comment:
Nice catch on the `user_ip`! One question though: not sure that we have access to `current_login_ip` before the record is saved, since it's a column that authlogic is managing. It's probably worth putting in a request spec showing that a request IP makes it through to the akismet call, since the IP is the #1 piece of info being used to judge spamminess and we had trouble with it before.

I think we'll need to add keys like `content` here which can be `profile.bio` and `permalink`

Reply:
Great piece of info. I'll confirm

Reply:
You are correct, how about passing it from the [controller](https://github.com/sudara/alonetone/pull/427/files#diff-4e05ad0d64e6100656b63ad1e78f32c5R123)?
```
  def user_params_with_ip
    user_params.merge(current_login_ip: request.remote_ip)
  end
```